lib/MusicBrainz/Server/Data/Search.pm

```diff
@@ -740,6 +745,19 @@ sub schema_fixup
         $data->{'artist_credit'} = MusicBrainz::Server::Entity::ArtistCredit->new( { names => \@credits } );
     }
 
+    if (defined $data->{aliases}) {
+        my @aliases = map {
+            MusicBrainz::Server::Entity::Alias->new(
```

Comment:
This seems to work, but please let me know if I should be doing something else to create these objects.